Processes/Docs-Contribution-Process-Internal.md

```diff
@@ -0,0 +1,105 @@
+
+The ENF Developer Relations team (DevRel) developed a set of templates to assist development of documentation for the EOS ecosystem. The templates are written in Markdown (.md) format and provide a structure for your content contribution. Take advantage of these writing tools to assist with your content contribution. These templates are guidelines for various types of content and will help you organize your content with a common look and feel. These templates are not a substitute for content accuracy and completeness.
+Complete the following steps to apply a template to your content:
+1. Open a new document in your GitHub branch web editor
+2. Open the DevRel repository **Template** Folder
+3. Open the template file for your document type. 
+
+      a. Overview  
+      b. Explainer  
+      c. Tutorial.  
+      d. How To
+4. Copy the contents of the template file
+5. Paste the contents of the template file into your new document in your GitHub branch web editor
+
+   The template provides a structure for your content
+6. Write your content
+
+   Use the topic designations provided by the template to organize your content  
+
+7. Submit the document containing your content to DevRel
+
+## Submit Content for Inclusion in EOS Documentation Portal
+
+**Scenario**: You have written content that needs to be added to the EOS Documentation portal It is in .md format. You wrote the content using HackMD, or the GitHub Web Editor, or in Google Docs and converted it to .md format. Now what do you need to do?
+
+Complete the following steps to submit your content to DevRel:
+
+1. Fill out the submission form with:
+* Your Name
+* Title of document
+* Document type
+  * Overview
+  * Explainer
+  * Tutorial
+  * How To
+* What role the document pertains to
+  * App Developer
+  * Node Operator
+  * Smart Contract Developer
+* Considerations
+  * Considerations can be special instructions to DevRel, an indication that the content will need more information, request to be uploaded to the EOS Documentation portal by a specific date, needs a review by other engineers, …
+
+2. Upload your titled .md formatted document and the submission form to the DevRel repository “Submit Content” folder
+
+   The DevRel team will refine and test the content. The DevRel team will take into account your considerations and contact you if they have any questions. They will also contact you for the final review and approval/signoff.
+
+## Submit Changes to Content in the EOS Documentation Portal
+**Note**: Changes to Content in the EOS Documentation Portal can include changes to content you have written or for other content.
+
+Complete the following steps to submit changes to content in the EOS Documentation Portal:
+1. Create a GitHub ticket/issue in the DevRel repository
+2. Include the following information in the ticket/issue:
+   * Name of the document that needs to be changed
+   * A description and/or list of changes that need to be made
+   * Priority
```

Comment:
This is just a tag, and should not be set by ticket creator

Reply:
Done